gradle.properties

```diff
@@ -12,7 +12,8 @@ version=3.0.0-rc-1-SNAPSHOT
 org.gradle.daemon=true
 org.gradle.caching=true
 org.gradle.vfs.watch=true
-org.gradle.jvmargs=-Xmx6g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
+org.gradle.configureondemand=true
```

Comment:
we need to watch this carefully, it can cause side effects

Reply:
I can revert the gradle changes here, I was just trying stuff to see if anything would stick before noticing it was always failing at the same test.